rpp30-boc-heartbleed/whimsical
# Test auth

## Description
configured client side testing. drafted initial test suites for login and register components with minimum of 60% code coverage.

## How to test
navigate to client folder. run npm i. run npm test. test script should run and display results and code coverage report. 100% of tests should pass and report should show about 66% code cover overall and for Login and Register components. 

## Notes
Jest must be v26, not 27.

## Issue tracking
https://trello.com/c/Qo776MSx/40-authentication-test-suites
